source/partials/_site_footer.html.erb

```diff
@@ -80,6 +80,12 @@
           Pricing
         </a>
       </li>
+
+      <li role="menuitem">
+        <a href="<%= localize_url('reviews', locale_id: locale_obj[:id]) %>" class="footer__link" title="<%= locale_page(page: 'reviews')[:page_title] %>">
```

Comment:
I don't believe we will have a `locale_page(page: 'reviews')[:page_title]`.  This likely needs its own title.